tools/ci/setup_js_env.sh

```diff
@@ -20,7 +20,7 @@ else
 
   # fail the whole test suite if bower install failed
   [ $STATUS = 0 ] || exit 1
-  [ -d vendor/assets/bower/bower_components ] || exit 1
+  #[ -d vendor/assets/bower/bower_components ] || exit 1
```

Comment:
(Removed properly in https://github.com/ManageIQ/manageiq-ui-classic/pull/4735/files#diff-df544bbd5873f755dd4d52d7a6bad0dfL23)